src/components/calcite-panel/resources.ts

```diff
@@ -22,6 +22,7 @@ export const ICONS = {
 
 export const SLOTS = {
   headerContent: "header-content",
+  headerSummary: "header-summary",
```

Comment:
@asangma can we remove this? I don't think its being used.

Reply:
Done.